lib/engine.py

```diff
@@ -290,19 +291,19 @@ def create_db(self):
         engine.db"""
         db_name = self.database_name()
         if db_name:
-            print "Creating database " + db_name + "..."
+            print("Creating database " + db_name + "...")
             # Create the database
             create_stmt = self.create_db_statement()
             if self.debug:
-                print create_stmt
+                print(create_stmt)
             try:
                 self.execute(create_stmt)
             except Exception as e:
                 try:
                     self.connection.rollback()
                 except:
                     pass
-                print "Couldn't create database (%s). Trying to continue anyway." % e
+                print("Couldn't create database (%s). Trying to continue anyway." % e)
```

Comment:
I agree with @goelakash that it's not necessary to make those changes in existing lines that we're touching to make Python 3 compatibility changes. It would be fine to make those changes at the same time, but I also don't think it hurts anything to wait. Any new lines that get added should use `"{}".format` should use.